src/Middleware/LoginSessionController.php

```diff
@@ -5,21 +5,14 @@
 use SilverStripe\Admin\LeftAndMain;
 use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Control\HTTPResponse;
-use SilverStripe\Core\Injector\Injector;
-use SilverStripe\ORM\ValidationException;
-use SilverStripe\Security\Member;
-use SilverStripe\Security\Security;
 use SilverStripe\Security\SecurityToken;
 use SilverStripe\SessionManager\Model\LoginSession;
 
-/**
- * Class LoginSessionController
- * @package SilverStripe\SessionManager\Control
- */
 class LoginSessionController extends LeftAndMain
 {
     private static $url_segment = 'loginsession';
 
+    // TODO: this is probably a legacy config that's not used anywhere so should be deleted
```

Comment:
Nope, without it, this will show up in the sidebar.

Reply:
Ah OK, it's because this extends LeftAndMain instead of Controller

I've updated the TODOs to change what this inherits from, and also move the folder

I'd do it on this PR, except I don't want to mix up features + refactoring on the same PR, and it may cause merge conflicts on other open PR's.  I've created a new issue to track this https://github.com/silverstripe/silverstripe-session-manager/issues/42